spec/features/spo_handover_cases_feature_spec.rb

```diff
@@ -0,0 +1,58 @@
+require "rails_helper"
+
+feature "SPO viewing upcoming handover cases" do
+  let(:prison) { 'LEI' }
+
+  context 'when signed in as an SPO', vcr: { cassette_name: :spo_handover_cases_feature } do
+    before do
+      allow_any_instance_of(Nomis::OffenderBase).to receive_messages(
```

Comment:
Overriding constants is nasty, but I think this is the best way to do this given the context. Nice find!